File: Iniciante/1005/1005.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner i = new Scanner(System.in);
        Double a = i.nextDouble();
        Double b = i.nextDouble();
        Double media = (3.5*a + 7.5*b)/11;
        System.out.printf("MEDIA = %.5f\n", media);
    }
 
}

File: Iniciante/1005/1005.c
#include <stdio.h>

int Media(double a, double b, double *media){
    *media = (3.5*a+7.5*b)/11;
}

int main(){
    double a, b, media;
    scanf("%lf", &a);
    scanf("%lf", &b);
    Media(a,b, &media);
    printf("MEDIA = %.5lf\n", media);
    return 0;
}